page.markdown

```diff
@@ -0,0 +1,291 @@
+
+Now, the problem with Nette, just like any software is that it gets very old very fast. Do you remember the release of PHP 5.3 in June 2009? This "namespace" thing, what a cool feature, I really loved it. Now it's 2019 and this is the status quo. This is a natural process of life, or would you prefer PHP 5.3 to be the last PHP version ever?
+
+> "Once you stop learning, you start dying."
+> _Albert Einstein_
+
+PHP frameworks just like any software get old. There is always a better version of it somewhere, or there is another framework that follows more modern standards, has a clearer API, or trendy features and is easier always a better version of it somewhere, or there is another framework that follows more modern standards, has a clearer API, or trendy features and is easier to learn.
+
+> "The only thing that is constant is change."
+> _Heraclitus_
+
+For the purpose of this post and showing our migration path, For the purpose of this post and showing our migration path, I'll use Nette because that's **the first project we migrate under a certain time** a certatime**... I'll tell you later the exact number of days! Keep reading! Keep reading.
+
+Also, I'll use Symfony as a go-to project, because that's our personal preference. Replace it with your personal favorite framework. 
+
+Why do we like Symfony? It has amazing [PSR-4 autodiscovery](https://www.tomasvotruba.cz/blog/2018/12/27/how-to-convert-all-your-symfony-service-configs-to-autodiscovery/#3-steps-to-your-minimalistic-configs) that makes your config 5-lines long, it's [6-months release cycle](https://www.tomasvotruba.cz/blog/2017/10/30/what-can-you-learn-from-menstruation-and-symfony-releases/) gets me exited all the time and it has great community that sometimes feels like my own family. 
+
+## What We Did?
+
+Now that we know that all software is dying, and the natural process of the software life-cycle, let's get technical!
+
+We migrate [Entry.do](https://entry.do/) project - API application built on controllers, routing, Kdyby integrations of Symfony and Doctrine. The application has been running in production for the last 4 years. We migrated it from Nette 2.4 to Symfony 4.2.
+
+![Figure 1: Entrydo website](figures/figures1.png)
+
+### How big is the Project?
+
+Honestly, with pattern refactoring the size doesn't matter, but people still tend to care about it. Quantitative numbers can be used to derive qualitative statements since for example size and complexity can behave orthogonal to each other. Put simply, something small can be very complex and something big does not have to be complex at all.
+
+If we don't count tests, migration, fixtures, etc., the application has **270 PHP files** totalling **54 357 lines** (without comments, using [phploc](https://github.com/sebastianbergmann/phploc)).
```

Comment:
```suggestion
If we don't count tests, migration, fixtures, etc., the application has ** 1186 PHP files**,  **1129 classes**, totalling **54 357 lines** (without comments, using [phploc](https://github.com/sebastianbergmann/phploc)).
```

Reply:
Thank you! 
I looked at directories, my bad